Microsoft.HealthVault.Fhir/Constants/FhirCategories.cs

```diff
@@ -20,7 +20,7 @@ public static class FhirCategories
         public static CodeableConcept VitalSigns = new CodeableConcept()
         {
             Text = "Vital Signs",
-            Coding = new List<Coding>() { ObservationCategoryCodes.VitalSignsCode }
+            Coding = new List<Coding>() { ObservationCategoryCodes.s_vitalSignsCode }
```

Comment:
I turned it public, it makes more sense actually

---
In reply to: [125131646](https://github.com/Microsoft/healthvault-fhir-library/pull/3#discussion_r125131646) [](ancestors = 125131646)